src/app/meta-repository.service.ts

```diff
@@ -98,7 +99,13 @@ export class MetadataRepositoryService {
             this.builder.buildThemeBootstrap(currentTheme, bootstrapData, bootstrapVersion) :
             this.builder.buildTheme(currentTheme, false, null, this.modifiedMetaCollection);
 
+        const savedBuildNumber = ++this.globalBuildNumber;
+
         return buildResult.then((result) => {
+            this.loading.hide();
```

Comment:
Shouldn't we hide a loading indicator only when it's the last build?

Reply:
No this.loading is a service with stack. We should call hide as many times as we call show.

Reply:
Sorry, I haven't noticed we have a Loading service and thought it's just hiding of a loading indicator.